docs/pxe/dgxie.md

```diff
@@ -9,7 +9,7 @@ You will need to download the official DGX Base OS ISO image to your provisionin
 Copy the DGX Base OS ISO to shared storage via a container running in Kubernetes, substituting the path to the DGX ISO you downloaded (be sure to wait for the `iso-loader` POD to be in the *Running* state before attempting to copy the ISO):
```

Comment:
I think we should reference the dgxie-container.md in a few new place. That README is a bit simpler to follow and more up-to-date

Reply:
Also, dgxie-container.md has a typo in containers/dgxie/docker-compose-yml.

I think we should merge the README with the dgxie-container by just chopping out step 3 and above from the current README and putting that at the top of the dgxie-container.md

Reply:
@supertetelman do you mind opening an issue for this or sending me an email? leaving it for another PR